RELEASENOTES.md

```diff
@@ -20,6 +20,8 @@
 -->
 # Cordova-node-xcode Release Notes
 
+* Updated `objects = {` to be followed by a blank line, matching current Xcode layout ([#42](https://github.com/apache/cordova-node-xcode/pull/46))
```

Comment:
We generally update RELEASENOTES.md when we make a release, as we documented in: <https://github.com/apache/cordova-coho/blob/master/docs/tools-release-process.md#update-release-notes--version>

Reply:
I’m sorry! I put a placeholder entry above the release line, happy to delete it if that’s less convenient or off process, here!

```suggestion
```

Reply:
Alternatively, this suggestion fixes the PR number, haha
```suggestion
* Updated `objects = {` to be followed by a blank line, matching current Xcode layout ([#46](https://github.com/apache/cordova-node-xcode/pull/46))
```